metecho/context_processors.py

```diff
@@ -10,6 +10,7 @@ def env(request):
     )
     GLOBALS = {
         "SITE": site_profile.data,
+        "GITHUB_ISSUE_LIMIT": settings.GITHUB_ISSUE_LIMIT,
```

Comment:
@jgerigmeyer @oluoluoxenfree I exposed this setting to the JS globals so you can read it in the frontend. You can use this value in the string that says `showing the first ${GITHUB_ISSUE_LIMIT} issues`.